cluster/mesos/docker/docker-compose.yml

```diff
@@ -149,9 +156,29 @@ scheduler:
     --cluster-dns=10.10.10.10
     --cluster-domain=cluster.local
     --v=2
+  environment:
+  - MESOS_DOCKER_ETCD_TIMEOUT
+  - MESOS_DOCKER_MESOS_TIMEOUT
+  - MESOS_DOCKER_API_TIMEOUT
   links:
   - etcd
   - mesosmaster1
   - mesosslave1
   - mesosslave2
   - apiserver
+sslgen:
```

Comment:
funny to see this. I had the same approach before moving everything into util.sh.